### PR TITLE
Added Data model Job & User

### PR DESCRIPTION
**Pull request by Eukst**
added bootstrap
added user model
added job model

### CODE REVIEW

File: test/fixtures/jobs.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```
Comment (mitaligaur) [inline test/fixtures/jobs.yml]: Remove this

File: test/models/job_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class JobTest < ActiveSupport::TestCase
```
Comment (mitaligaur) [inline test/models/job_test.rb:5]: Add comments

File: app/models/job.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class Job < ApplicationRecord
```
Comment (mitaligaur) [inline app/models/job.rb:4]: Add comments

**Review by mitaligaur — CHANGES_REQUESTED** [review]

File: test/fixtures/jobs.yml (hunk 1 above)
Comment (Eukst) [reply test/fixtures/jobs.yml]: done

File: test/models/job_test.rb (hunk 2 above)
Comment (Eukst) [reply test/models/job_test.rb:5]: done

File: app/models/job.rb (hunk 3 above)
Comment (Eukst) [reply app/models/job.rb:4]: done

File: app/models/user.rb [hunk 4]
```diff
@@ -2,8 +2,21 @@
...
+  has_many:jobs , dependent: :destroy
   # Include default devise modules. Others available are:
   # , :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  # validation
+  VALID_PIN_REGEX = /[1-9]{1}[0-9]{5}/
+
+  validates :first_name, presence: true
```
Comment (Sharee-ongraph) [inline app/models/user.rb]: no need to write presence:true  for each field separately, 
validates: fields name separated by comma, presence:true

**Review by Sharee-ongraph — APPROVED** [review]

File: app/models/user.rb (hunk 4 above)
Comment (Eukst) [reply app/models/user.rb]: done

**Review by mitaligaur — APPROVED** [review]

**Pull request merged**